src/utils/multisigHelpers.ts

```diff
@@ -0,0 +1,166 @@
+import { SubstrateEvent } from "@subql/types";
+import { u8aToHex } from "@polkadot/util";
+import { CreateCallVisitorBuilder, CreateCallWalk } from "subquery-call-visitor";
+import { EventStatus, MultisigEvent, MultisigOperation, OperationStatus } from "../types";
+import { AccountId, DispatchResult, Timepoint } from "@polkadot/types/interfaces";
+import { 
+  generateEventId, 
+  getBlockCreated, 
+  getDataFromCall, 
+  getDataFromEvent, 
+  getIndexCreated, 
+  timestamp 
+} from "./operations";
+
+const callWalk = CreateCallWalk();
+
+/**
+ * Detects if the extrinsic contains an asMultiThreshold1 call using the visitor pattern
+ * @param event - The Substrate event
+ * @returns True if the extrinsic contains an asMultiThreshold1 call, false otherwise
+ */
+export async function isThreshold1(event: SubstrateEvent): Promise<boolean> {
```

Comment:
define isThreshold1, not ideal, but it is tricky to define which extrinsic has produced threshold1 event. Please suggest better solution if it has